engine/access/rest/accounts_test.go

```diff
@@ -28,77 +33,76 @@ func accountURL(address string, height string) string {
 func TestGetAccount(t *testing.T) {
 	backend := &mock.API{}
 
-	t.Run("get by address at latest block", func(t *testing.T) {
-		heights := []string{"", "latest"}
+	t.Run("get by address at latest sealed block", func(t *testing.T) {
```

Comment:
the height can be `final` or `sealed` (not `latest`). Added two unittest cases accordingly

Reply:
Good that you fixed this but in general I also really think we should support `latest` alias, I've pinged Simon about it to open a discussion. We don't have any documentation about statuses anywhere and developers usually know what latest means whereas final and sealed is really technical. Even to me I have to think couple of times which one means latest.